src/foundry/client/canvas/sources/point-vision-source.d.mts

```diff
@@ -239,21 +327,30 @@ declare namespace PointVisionSource {
   }
 
   /** @privateRemarks Foundry types this as just the Record, but only ever checks the one key */
-  interface BlindedReasons extends Record<string, boolean | undefined> {
+  interface BlindedReasons extends Record<string, boolean> {
```

Comment:
Yeah so it was `boolean | undefined` for a reason; specifically you can see all the errors below. Locally we run with `exactOptionalPropertyTypes` on but I have CI run twice, once for each, as there's errors exclusive to one or the other. In this case `darkness?: boolean` is a valid item for `Record<string, boolean>` only with `exactOptionalPropertyTypes` on. Fun right?